news/2019-04-10-bn-and-qat-restructure.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "BN and QAT Restructure"
+date: 2019-05-01 12:00:00 +0000
+---
+
+Changes to the ranking process are nigh! With recent discussions about [changes to the Quality Assurance Team and Beatmap Nominators](https://osu.ppy.sh/community/forums/topics/862002), we are proud to present to you the following changes.
+
+![](/wiki/shared/news/banners/beatmap-management.jpg)
+
++ The QAT is gone and has been replaced by the new [Nominator Administration Team](https://osu.ppy.sh/groups/7).
++ The Beatmap Nominators (BN) are responsible for disqualifying beatmaps and mediating vetoes from now on, in addition to their task of pushing beatmaps towards the ranked category.
++ The Nominator Administration Team (NAT) are the moderators of the Beatmap Nominators. They are responsible for BN Applications, creating tests for the applicants and for evaluating the current members. They will still have their moderation privileges although it is not their primary focus. 
++ Beatmaps that have been vetoed will be mediated by a random jury made up of members of the BN. The NAT will oversee these mediations. 
++ The Global Moderation Team (GMT) will now handle beatmap moderation, and as such will also have access to the disqualify button. The GMT will only disqualify beatmaps in cases where they contain inappropriate content.
++ The current Beatmap Nominator usergroup will be used for probationary Beatmap Nominators. A new usergroup will be made for regular members of the Beatmap Nominators which will grant access to the disqualify button.
+
+The primary focus of probationary BNs will be to nominate beatmaps, and so will not have the ability to disqualify beatmaps or take part in mediating vetoes. Moreover, they will be unable to nominate beatmaps with more than one game mode, and a beatmap cannot be qualified by two probationary BNs. New members of the BNs will be placed in probation for approximately one month before their performance is reviewed by the NAT. They will then be promoted to a regular BN, remain in probation, or be removed from the group completely. Members of the regular BNs may also be demoted to probation if their work is found to not meet standards.
+
+The [Beatmap Nominator rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules) have been updated accordingly for their new tasks. Moreover, work on a score system as a guideline to evaluate Beatmap Nominators starts now and you can expect a new proposal detailing our plans for that soon!
+
+Additionally to this, BN applications will be open indefinitely from the time of this news post! We have a [new website](http://bn.mappersguild.com/) to deal with BN affairs including applications and reporting BNs. Special thanks to [pishifat](https://osu.ppy.sh/users/3178418) and [Milan-](https://osu.ppy.sh/users/1052994) for developing the website. 
```

Comment:
mapping-related reports in general, not just reporting BNs (stolen maps, inappropriate content, and bn/nat conduct)

also last sentence credits dont need to exist